frontend/src/components/UI/ModeIndicator.module.scss

```diff
@@ -1,5 +1,6 @@
 .Indicator {
   position: fixed;
+  // TODO: move all z-index numbers into a css constant file to avoid increasingly giant numbers like this.
```

Comment:
Maybe we can file a GitHub issue for this? 👀🌾🚜

Reply:
https://github.com/cornell-dti/samwise/issues/532